packages/react-ui-core/src/Card/__tests__/Card-test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import theme from './mocks/theme'
+import ThemedCard from '../Card'
+
+const Card = ThemedCard.WrappedComponent
+const content = <h1>Hello world</h1>
+
+describe('Card', () => {
+  let wrapper
+
+  beforeEach(() => {
+    wrapper = shallow(
+      <Card />,
+      { lifecycleExperimental: true }
```

Comment:
@mneumark found based on our enzyme version we actually don't need this. So we can remove it 😄